packages/vats/src/proposals/probeZcfBundle.js

```diff
@@ -0,0 +1,72 @@
+import { E } from '@endo/far';
+import { makeStorageNodeChild } from '@agoric/internal/src/lib-chainStorage.js';
+
+// verify that Zoe remembers the zcfBundleCap across upgrades
+export const probeZcfBundleCap = async (
+  {
+    consume: {
+      vatAdminSvc,
+      walletFactoryStartResult,
+      provisionPoolStartResult,
+      chainStorage,
+      walletBridgeManager: walletBridgeManagerP,
+      vatStore,
+    },
+  },
+  options,
+) => {
+  const { zoeRef, zcfRef, walletRef } = options.options;
+
+  const { adminNode, root: zoeRoot } = await E(vatStore).get('zoe');
+  const zoeConfigFacet = await E(zoeRoot).getZoeConfigFacet();
+
+  // STEP 1: restart WF; it'll use the newest ZCF //////////////////
```

Comment:
`newest` is ambiguous here, as it's not the zcf bundle that is being installed just below.

Reply:
yeah, just the most recent known to Zoe. updated.